.github/workflows/build_appstore.yml

```diff
@@ -102,9 +105,6 @@ jobs:
     - name: Select Xcode
       run: sudo xcode-select -s /Applications/Xcode_$(<.xcode-version).app/Contents/Developer
 
-    - name: Prepare fastlane
```

Comment:
If a fastlane step existed in a workflow, I've now moved it to the next step after code checkout – in case we add more fastlane-dependent steps later.